gapic-generator/lib/gapic/presenters/method_rest_presenter.rb

```diff
@@ -189,6 +189,24 @@ def query_string_params_var_name
       def transcoding_helper_name
         "transcode_#{@main_method.name}"
       end
+
+      ##
+      # Method name
+      #
+      # @return [String]
+      #
+      def name
+        @main_method.name
+      end
```

Comment:
Nit: replace `@main_method.name`, e.g. in `transcoding_helper_name` with this

Reply:
done